maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,737 @@
+except ImportError:
+    PYDANTIC_AI_AVAILABLE = False
+    Agent = None
+    AbstractAgent = None
+    Tool = None
+    Model = None
+    AgentRun = None
+    ModelRequestNode = None
+    CallToolsNode = None
+
+from ...logger import (
+    Generation,
```

Ruff (0.12.2)

`...logger.Generation` imported but unused

Remove unused import

(F401)